internal/init/init.go

```diff
@@ -85,40 +86,31 @@ func (c *Client) NewCommand() *cobra.Command {
 			fmt.Println()
 			token := string(b)
 			c.Servicer.SetToken(token)
-			metalClient := c.Servicer.API(cmd)
-			c.UserService = metalClient.Users
-			c.ProjectService = metalClient.Projects
+			metalGoClient := c.Servicer.MetalAPI(cmd)
+			c.UserService = *metalGoClient.UsersApi
+			c.ProjectService = *metalGoClient.ProjectsApi
 
-			user, _, err := c.UserService.Current()
+			user, _, err := c.UserService.FindCurrentUser(context.Background()).Execute()
 			if err != nil {
 				return err
 			}
-			organization := user.DefaultOrganizationID
-			project := ""
-			if user.DefaultProjectID != nil {
-				project = *user.DefaultProjectID
-			}
+			organization := user.AdditionalProperties["default_organization_id"]
+
 			fmt.Printf("Organization ID [%s]: ", organization)
 
 			userOrg := ""
 			fmt.Scanln(&userOrg)
 			if userOrg == "" {
-				userOrg = organization
+				userOrg = fmt.Sprintf("value: %v", organization)
```

Comment:
Is the `value: ` prefix needed?  I would expect this to break anything that uses the config file.

Reply:
yes, ```value:``` is not needed, but ```metal init``` is going generate/write config file

Reply:
The `value: ` prefix should be removed so that the config file is valid
```suggestion
				userOrg = organization
```

Reply:
It is shows following error "cannot use organization (variable of type interface{}) as string value in assignment: need type assertion"